scripts/i18n_utility/README.md

```diff
@@ -0,0 +1,106 @@
+# I18n Utility Script
+
+This is a CLI utility for:
+* identifying orphaned translation related files
+* deleting orphaned files
+* identifying rename changes to make for translation related files
+* making rename changes for translation related files
+
+Orphaned files refer to files that exist underneath `i18n/content/docs` that do not have an english equivalent and are therefore unused and safe to delete.
+
+For the rename changes, this refers to when you move an english document and don't move an existing translated version of that doc. So if we have a doc `hello/world.txt` and we rename it to `hello/goodbye.txt` we also want to rename the translated versions of that document (if they exist). The last two commands will print out subsequent rename changes that need to be made (based on what has been renamed already in the english content directory) and make those changes for you.
+
+## Important Notes
+
+There are a couple scenarios that are easy enough to find yourself in where you may not see output that you expect to, or you think the script isn't running correctly.
+
+### Scenario: Branch is not up to date
+
+It may be the case that you see a (rename) change listed that you did not make. The script runs against `origin/develop` and creates a diff between the two branches. This can include changes that do not exist locally if the branch is behind develop. The more behind a branch is, the higher the chance of running into this is.
+
+The fix: keep working branch up to date. Merge or rebase with develop frequently.
+
+### Scenario: No renames detected
+
+If files are manually being moved around, there is a chance that git does not detect that as a rename operation. Instead, git may initially see this as having created & deleted two files. If the script is run in this instance, renamed files will not be detected.
```

Comment:
Personally, it'd be great to have a better understanding of why this happens (changes not being registered by git).

Reply:
if the explanation below is good to you, i can add that to the readme.